Userland/Libraries/LibDeviceTree/FlattenedDeviceTree.h

```diff
@@ -37,4 +41,23 @@ struct FlattenedDeviceTreeReserveEntry {
 };
 static_assert(sizeof(FlattenedDeviceTreeReserveEntry) == 16, "FDT Memory Reservation entry size must match specification");
 
+// https://devicetree-specification.readthedocs.io/en/v0.3/flattened-format.html#lexical-structure
+enum FlattenedDeviceTreeTokenType : u32 {
```

Comment:
`enum class`

Reply:
Because we always read this from a BigEndian source and aren't sure if it's actually going to be in bounds for that enum, having it be a non-classed enum makes the big switch statement in `walk-device_tree` a lot simpler (and easier to read).